package.json

```diff
@@ -24,13 +24,14 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/utils": "^5.57.0",
+    "@typescript-eslint/utils": "^6.0.0",
     "tslib": "^2.3.1",
-    "tsutils": "^3.21.0"
+    "tsutils": "^3.21.0",
+    "typescript-eslint-utils-5": "npm:@typescript-eslint/utils@^5.62.0"
```

Comment:
I do not think it's a good idea to have this package all the time just to test for BC.
You can just install it in CI for the BC check and the test code that you've added should still work fine.

Reply:
Oops, I also don't meant for that 😬

Was planning on just installing it as part of CI!